cmd/edit/cluster/cmd.go

```diff
@@ -463,6 +545,98 @@ func run(cmd *cobra.Command, _ []string) {
 		}
 	}
 
+	/*******  Registries Config *******/
+	updateRegistriesConfig := insecureRegistries != nil ||
+		blockedRegistries != nil ||
+		allowedRegistriesForImport != "" ||
+		allowedRegistries != nil ||
+		platformAllowlist != "" ||
+		additionalTrustedCa != ""
+
+	if !updateRegistriesConfig && interactive.Enabled() {
+		updateRegistriesConfigValue, err := interactive.GetBool(interactive.Input{
+			Question: "Update registries config",
+			Default:  updateRegistriesConfig,
+		})
+		if err != nil {
+			r.Reporter.Errorf("Expected a valid value: %s", err)
+			os.Exit(1)
+		}
+		updateRegistriesConfig = updateRegistriesConfigValue
+	}
+
+	if updateRegistriesConfig && interactive.Enabled() {
+		allowedRegistriesInputs, err := interactive.GetString(interactive.Input{
+			Question: "Allowed Registries",
+			Help:     cmd.Flags().Lookup("allowed-registries").Usage,
+			Default:  allowedRegistries,
+		})
+		if err != nil {
+			r.Reporter.Errorf("expected a valid value for allowed registries: %s", err)
+			os.Exit(1)
+		}
+		allowedRegistries = helper.HandleEmptyStringOnSlice(strings.Split(allowedRegistriesInputs, ","))
+
+		blockedRegistriesInputs, err := interactive.GetString(interactive.Input{
+			Question: "Blocked Registries",
+			Help:     cmd.Flags().Lookup("blocked-registries").Usage,
+			Default:  blockedRegistries,
+		})
+		if err != nil {
+			r.Reporter.Errorf("expected a valid value for blocked registries: %s", err)
+			os.Exit(1)
+		}
+		blockedRegistries = helper.HandleEmptyStringOnSlice(strings.Split(blockedRegistriesInputs, ","))
+
+		insecureRegistriesInputs, err := interactive.GetString(interactive.Input{
+			Question: "Insecure Registries",
+			Help:     cmd.Flags().Lookup("insecure-registries").Usage,
+			Default:  insecureRegistries,
+		})
+		if err != nil {
+			r.Reporter.Errorf("expected a valid value for insecure registries: %s", err)
+			os.Exit(1)
+		}
+		insecureRegistries = helper.HandleEmptyStringOnSlice(strings.Split(insecureRegistriesInputs, ","))
+
+		allowedRegistriesForImport, err = interactive.GetString(interactive.Input{
+			Question: "Allowed Registries For Import",
+			Help:     cmd.Flags().Lookup("allowed-registries-for-import").Usage,
+			Default:  allowedRegistriesForImport,
+			Validators: []interactive.Validator{
+				ocm.ValidateAllowedRegistriesForImport,
+			},
+		})
+		if err != nil {
+			r.Reporter.Errorf("expected a valid value for allowed registries for import: %s", err)
+			os.Exit(1)
+		}
+
+		platformAllowlist, err = interactive.GetString(interactive.Input{
```

Comment:
I think we should remove this from interactive flow, let's just leave it as hidden flag.
Reason is that this is a specific flow in case platform registries change, otherwise user doesn't need to care about it.

Reply:
Removed from interactive mode